db/revert/tables/analyst.sql

```diff
@@ -1,7 +1,45 @@
--- Revert ccbc:tables/analyst from pg
+-- Deploy ccbc:tables/analyst to pg
 
 begin;
 
-drop table ccbc_public.analyst;
+create table if not exists ccbc_public.analyst(
```

Comment:
I think this should be undone, looks like there's a rework that was done but not reverted